### PR TITLE
Added Tailwind, big refactor

### PR DESCRIPTION
**Pull request by tellaho**
image

- **Added Tailwind**: I added the recommended tailwind vscode extension, which adds intellisense. Adding the `prettier-plugin-tailwindcss` Prettier package automatically turns it on.

- **New homepage**: Visual design and interaction pass on the homepage. Playing around with using a separate, contrast-y visual for the global selected pills.

- **Refactored all the things:** Refactored _a lot_ of our UI to use tailwind class names. Their docs are actually super useful (I started using cmd + k really good by the end of this 😆

- **Well, nearly all the things:** Chose to delete unused code instead of refactor, including BannerAlert, etc. The goal is to eventually migrate some components into the [shadcn/ui](https://ui.shadc.com/docs/) library.

### Test case:
- Open [homepage](https://hawaiiansintech-4t97w7cvd-hawaiians.vercel.app/) and play with filtering.
- Go through Join and Request Changes flow.
- Read About page.
- ~Thank you and Hackathon pages still needs refactor~

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -108,7 +107,7 @@ export default function HomePage({
   const [membersCount, setMembersCount] = useState<number>(
     initialState.members.length
   );
-  const [viewAll, setViewAll] = useState<Boolean>(true);
```
Comment (eparubrub) [inline pages/index.tsx:111]: oh woops hope that wasn't me, just read about the difference between these - https://stackoverflow.com/a/64443353

File: pages/join/03-company.tsx [hunk 2]
```diff
@@ -161,156 +160,146 @@ export default function JoinStep3({ industries, pageTitle }) {
...
-            ? WorkExperienceWarning()
-            : null}
+              ) : (
+                <Input
+                  autoFocus
+                  name="add-field"
+                  centered
+                  fullHeight
+                  onChange={(e) => {
+                    setIndustrySuggested(e.target.value + "fda");
```
Comment (eparubrub) [inline pages/join/03-company.tsx]: errrr is this `fda` thing supposed to be here? it's adding this unexpected text when I'm typing "test"

Jun-30-2023 18-02-50

**Review by eparubrub — CHANGES_REQUESTED** [review]
Looked over everything, most if not all changes are UI specific and not changing any functional/data stuff. I really like the updates, cleans up a lot of the <style jsx> tags that we have

I tested out the join workflows and everything seemed to be in order and reflecting on the backside. The only real issue I found was the `fda` text I mentioned below, and the UI on the iphone seems to be a bit laggy/jittery, but that might be because of the vercel hosting, will send my screen cap on discord. But I think after that `fda` fix we're good.

Also, thanks for the updates, I can't imagine how much time you put into this refactor

File: pages/index.tsx (hunk 1 above)
Comment (tellaho) [reply pages/index.tsx:111]: All good. Got me to look for others, found a couple more. Fixed in a commit.

**Review by eparubrub — APPROVED** [review]

**Comment by tellaho** [comment]
- **Fixed `fda` bug.** Prob was testing something and forgot.
- **Removed `MemberDirectory` animation altogether.** It was starting to bug once I converted it to the grid layout (versus the old table rows).
- **Replaced all `Boolean` → `boolean`.**
- **Fixed an issue where the "dot" wasn't showing on `Checkbox`/`Selectable`/`RadioBox` for smaller screens.** Noticed it in the screenshot you sent on Discord.

**Pull request merged**